MoonAngleCondition/MoonAngleCondition.cs

```diff
@@ -161,6 +173,25 @@ public class MoonAngleCondition : SequenceCondition, IValidatable {
             return i.Count == 0;
         }
 
+        private double LorentzianSeparation(double distance, int width) {
+            // Moon-Avoidance Lorentzian formulated by the Berkeley Automated Imaging Telescope (BAIT) team
+            // Formula borrowed from ACP http://bobdenny.com/ar/RefDocs/HelpFiles/ACPScheduler81Help/Constraints.htm
+            var observerInfo = new ObserverInfo() {
```

Comment:
Since we now have 2 places where we're creating an `ObserverInfo` object, you could break it off into its own private method and just call that from this and the plain function. Be sure to include the weatherdata stuff for refraction adjustments. Just copy the lines from `CalculateMoonTargetSeparation()` below.

Reply:
Done